src/models/accounts.rs

```diff
@@ -3,7 +3,8 @@ use bigdecimal::BigDecimal;
 use crate::schema;
 use schema::accounts;
 
-#[derive(Insertable, Debug, Clone)]
+#[table_name = "accounts"]
+#[derive(Insertable, Debug, Clone, QueryableByName)]
```

Comment:
```suggestion
#[derive(Insertable, Debug, Clone, QueryableByName)]
#[table_name = "accounts"]
```

`table_name` attribute becomes available with `QueryableByName` derive, so it makes sense to flip the order